pyproject.toml

```diff
@@ -1,10 +1,10 @@
-[build-system]
-requires = ["flit"]
-build-backend = "flit.buildapi"
-
-[tool.flit.metadata]
+[project]
 module = "orchestrator"
+name = "orchestrator"
 dist-name = "orchestrator-core"
+# Version is mandatory
+version = "0.3.4"
```

Comment:
`pyproject.toml` should be added to https://github.com/workfloworchestrator/orchestrator-core/blob/main/.bumpversion.cfg to keep this version in sync with the other files